specs/ReleaseChannelPreferenceSet.md

```diff
@@ -0,0 +1,325 @@
+            // `CoreWebView2RuntimeChannel`s. OR operation(s) can be applied to multiple
+            // `CoreWebView2RuntimeChannel`s to create a mask. . The default value is a
+            // a mask of all the channels. By default, the WebView2 loader searches for
+            // channels from most to least stable, using the first channel found on the
+            // device. When a `RuntimeChannelPreferenceSet` is provided, the WebView2
+            // loader will only search for the channels specified in the set.
+            // Set `ReverseChannelSearchOrder` to `true` to reverse the search order so
+            // that the loader searches for least stable build first. See
+            // `CoreWebView2RuntimeChannel` for descriptions of each channel. Environment creation
+            // fails if the loader is unable to find any channel from the
+            // `RuntimeChannelPreferenceSet` installed on the device. Use
+            // `CoreWebView2Environment.GetAvailableBrowserVersionStringWithOptions` to verify which channel
+            // is used.
+            //
+            // Examples:
+            // |   RuntimeChannelPreferenceSet   |   Default Channel Search Order   |   Reverse Channel Search Order   |
+            // | --- | --- | --- |
+            // |CoreWebView2RuntimeChannel.Beta \| CoreWebView2RuntimeChannel.Stable| WebView2 Runtime -> Beta | Beta -> WebView2 Runtime|
+            // |CoreWebView2RuntimeChannel.Canary \| CoreWebView2RuntimeChannel.Dev \| CoreWebView2RuntimeChannel.Beta \| CoreWebView2RuntimeChannel.Stable | WebView2 Runtime -> Beta -> Dev -> Canary | Canary -> Dev -> Beta -> WebView2 Runtime |
+            // |CoreWebView2RuntimeChannel.Canary| Canary | Canary |
+            // |CoreWebView2RuntimeChannel.Beta \| CoreWebView2RuntimeChannel.Canary \| CoreWebView2RuntimeChannel.Stable | WebView2 Runtime -> Beta -> Canary | Canary -> Beta -> WebView2 Runtime |
+            //
+            // If both a `BrowserExecutableFolder` and a `RuntimeChannelPreferenceSet` are
+            // provided, the `BrowserExecutableFolder` takes precedence. The
+            // `RuntimeChannelPreferenceSet` can be overridden by the corresponding registry
+            // override `RuntimeChannelPreferenceSet` or the environment variable
+            // `WEBVIEW2_RUNTIME_CHANNEL_PREFERENCE_SET`. Set the value to a comma-separated string of
+            // integers, which map to the `CoreWebView2RuntimeChannel` values:
+            // Stable (0), Beta (1), Dev (2), and Canary (3). For example, the value "0,2"
+            // indicates that the loader should only search for Dev channel and the WebView2
+            // Runtime, using the order indicated by `ShouldReverseChannelSearchOrder`.
+            // The loader attempts to interpret each integer and treats any invalid entry as
+            // Stable channel.
+            CoreWebView2RuntimeChannel RuntimeChannelPreferenceSet { get; set; };
```

Comment:
Do we still avoid using flags enums as the type of a property?  I don't recall if that guideline was primarily for JavaScript compatibility.

Reply:
This is effectively a write-only property even though it has a getter. The guideline was about being hard to read flags enums but we're mostly not doing that here.